src/frontend/hooks/server/invites.ts

```diff
@@ -21,23 +24,30 @@ export function usePendingInvites() {
 export function useAcceptInvite(projectId?: string) {
   const mapeoApi = useApi();
   const queryClient = useQueryClient();
-  const switchActiveProject = useUpdateActiveProjectId();
+  const switchActiveProject = usePersistedProjectId(
+    state => state.setProjectId,
+  );
 
   return useMutation({
     mutationFn: async ({inviteId}: {inviteId: string}) => {
       if (!inviteId) return;
-      mapeoApi.invite.accept({inviteId});
+      return mapeoApi.invite.accept({inviteId});
     },
     onSuccess: () => {
       // This is a workaround. There is a race condition where the project in not available when the invite is accepted. This is temporary and is currently being worked on.
       setTimeout(() => {
-        queryClient
```

Comment:
Wondering if the changes here might help with the comment above this call? wasn't too sure so didn't attempt to make any changes to fix it in this PR